docs/get-started/concepts/filtering.md

```
@@ -131,19 +131,19 @@ client.files.filesets.list(filter={"purpose": {"$eq": "dataset"}})
 
 ```python
 # Filter by a custom label
 benchmarks = client.evaluation.benchmarks.list(
-    extra_query={"filter[data.labels.eval_category]": "agentic"}
+    extra_query={"filter[labels.eval_category]": "agentic"}
 )
 ```
 
 The equivalent REST call:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)